app/templates/src/test/resources/config/_application.yml

```diff
@@ -11,6 +11,9 @@ async:
     maxPoolSize: 50
     queueCapacity: 10000
 
+liquibase:
+    context: test
```

Comment:
This is annoying: this is supposed to be the base configuration, so "test" isn't good. Can you remove those lines? They will be overriden by the "dev" or "prod" profiles